variants/PNUCLEO_WB55RG/variant.cpp

```diff
@@ -99,9 +100,15 @@ WEAK void SystemClock_Config(void)
   RCC_ClkInitTypeDef RCC_ClkInitStruct = {};
   RCC_PeriphCLKInitTypeDef PeriphClkInitStruct = {};
 
+  /* This prevents the CPU2 (M0+) to configure RCC */
```

Comment:
It does not prevent M0+ to configure RCC, it prevents M0+ to configure RCC at the same time. It can still happen later, after semaphore release.

Reply:
Cuopy from MBed clock config. 😜 
Anyway at this time when doing the lock the comment is valid while the lock is kept.
And of course release will allow M0+ to configure it.
What do you propose instead ?

Reply:
"This prevents concurrent access to RCC registers by CPU2 (M0+)"